index.js

```diff
@@ -160,21 +171,17 @@ function start_compile_server() {
 }
 
 function initialize_options(that) {
-    Owl.options = loaderUtils.getOptions(that);
-    if (typeof Owl.options.hmr === 'undefined') { Owl.options.hmr = false; }
-    if (typeof Owl.options.hmrHook === 'undefined') { Owl.options.hmrHook = ''; }
-    if (typeof Owl.options.sourceMap === 'undefined') { Owl.options.sourceMap = false; }
-    if (typeof Owl.options.includePaths === 'undefined') { Owl.options.includePaths = null; }
-    if (typeof Owl.options.requireModules === 'undefined') { Owl.options.requireModules = null; }
-    if (typeof Owl.options.dynamicRequireSeverity === 'undefined') { Owl.options.dynamicRequireSeverity = null; }
-    if (typeof Owl.options.compilerFlagsOn === 'undefined') { Owl.options.compilerFlagsOn = null; }
-    if (typeof Owl.options.compilerFlagsOff === 'undefined') { Owl.options.compilerFlagsOff = null; }
+    const options = loaderUtils.getOptions(that);
+    Object.keys(default_options).forEach(
+        (key) => if (option[key] === undefined) option[key] = default_options[key];
```

Comment:
must be options[key] and the whole thing after => in curly brackets, i fixed it.